feature/sessions/src/main/java/io/github/droidkaigi/confsched2023/sessions/component/TimetableRoom.kt

```diff
@@ -0,0 +1,26 @@
+package io.github.droidkaigi.confsched2023.sessions.component
+
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.graphics.Color
+import io.github.droidkaigi.confsched2023.designsystem.theme.hallColors
+import io.github.droidkaigi.confsched2023.model.RoomIndex.Room1
+import io.github.droidkaigi.confsched2023.model.RoomIndex.Room2
+import io.github.droidkaigi.confsched2023.model.RoomIndex.Room3
+import io.github.droidkaigi.confsched2023.model.RoomIndex.Room4
+import io.github.droidkaigi.confsched2023.model.RoomIndex.Room5
+import io.github.droidkaigi.confsched2023.model.TimetableRoom
+import io.github.droidkaigi.confsched2023.model.type
+
+internal val TimetableRoom.color: Color
+    @Composable get() {
+        val colors = hallColors()
+
+        return when (type) {
+            Room1 -> colors.hallC
+            Room2 -> colors.hallA
+            Room3 -> colors.hallB
+            Room4 -> colors.hallD
+            Room5 -> colors.hallE
```

Comment:
Looks like the mapping of a room to hall color token is not ordered (right?), so I believe we should have the logic in common instead of writing the same code ad hoc. We could have a better idea for this...

Reply:
Sorry for not informing you.
Yeah, we shouldn't use the sort property. We should use the name. Could you apply it to your logic?
https://github.com/DroidKaigi/conference-app-2023/issues/633

Reply:
Thank you for the information!

> Could you apply it to your logic?

Yep, sure! Then, I use `name` for the room type mapping. The code must be here: https://github.com/DroidKaigi/conference-app-2023/blob/d0800dd0ba35a837751c2890ac6cd2801dac2e5e/core/model/src/commonMain/kotlin/io/github/droidkaigi/confsched2023/model/TimetableRoom.kt#L11